css/properties/position.json

```diff
@@ -0,0 +1,178 @@
+{
+  "css": {
+    "properties": {
+      "position": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/position",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1",
+              "notes": [
+                "Since Firefox 30, Firefox allows <a href='https://developer.mozilla.org/docs/Web/HTML/Element/tr'><code>&lt;tr&gt;</code></a>, <a href='https://developer.mozilla.org/docs/Web/HTML/Element/thead'><code>&lt;thead&gt;</code></a>, and <a  href='https://developer.mozilla.org/docs/Web/HTML/Element/tfoot'><code>&lt;tfoot&gt;</code></a> elements with a <code>position: relative;</code> style to act as absolute positioning containers. This means that a <code>position: absolute;</code> styled element inside the table can be positioned relative to these elements. In other browsers and in older versions of Firefox, setting <code>position: relative;</code> on a table row or row group has no effect. Firefox helps developers transition to the new behavior and detect any rendering issues it may cause on their sites by printing the following warning to the JavaScript console: &quot;Relative positioning of table rows and row groups is now supported. This site may need to be updated because it may depend on this feature having no effect.&quot;",
```

Comment:
I think this should be a subfeature, not a note.

Reply:
OK, I'm adding a commit to do this, though I'm not exactly sure how to add data for the other browsers. Some searching to figure out what to call this feature led me to believe that [the original note wasn't true](https://stackoverflow.com/questions/5148041/does-firefox-support-position-relative-on-table-elements) and that this behavior was conventional if not standard on other browsers prior to FF 30 (it is part of the draft [CSS Positioned Layout Module Level 3](https://drafts.csswg.org/css-position-3/#position-property) however).